Assets/Scripts/GameManager.cs

```diff
@@ -75,4 +102,33 @@ private void RestartLevel()
         _UIManager.RestartLevelUIReset();
         LevelUpdate();
     }
+
+    private void Save()
+    {
+        PlayerPrefs.SetInt("savedLevel", _currentLevel);
```

Comment:
i would make the access strings for the PlayerPrefs constants, especially since they get reused. you can make them private constants that are accessible throughout the file, that way you don't have hardcoded "savedLevel" in multiple places.

this makes it so if you ever have to change the name, you would only need to do it in 1 place. it significantly lowers the chances of discrepancies or spelling errors that could break the code.